open-craft/ansible-playbooks
# Add initial Nomad role

This PR adds the relevant Nomad role, playbook, and common public variables needed to get a Nomad cluster running, including both the server and client nodes.

Test Instructions:

- Check certificates on any single Nomad server and client with `openssl x509 -noout -text -in cert.pem` -- ensure that the SAN list makes sense.
- On both a Nomad server and client, do `nomad server members` and ensure all Nomad servers for the cluster are there. Do `nomad node status` to see that all clients are there.
- Run the Nomad playbook afresh, without selecting any group -- see that it succeeds for all involved servers without issue.
